lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -531,11 +531,14 @@ module.exports = class MethodMappings {
           return `/element/${id}/location_in_view`;
         },
 
-        async isElementDisplayed(webElementOrId) {
+        isElementDisplayed(webElementOrId) {
+          if (this.transport.api.isAppiumClient()) {
```

Comment:
We should keep a helper method in mobile utils instead of using api commands over here for `isAppiumClient`. API command can then use that same helper function